types/jquery-countto/index.d.ts

```diff
@@ -35,7 +35,7 @@ export interface Options {
     /**
      * A handler that is used to format the current value before rendering to the DOM
      */
-    formatter: (value: number, options: Options) => string;
+    formatter?: (value: number, options: Options) => string;
```

Comment:
Nit: Maybe worth changing this whole interface to:
```typescript
export type Options = Partial<{
  from: number,
  to: number,
  speed: number,
  ...
}>;
```
so that future maintainers don't need to remember to add `?` to the keys?